lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -55,12 +55,8 @@ class FeatureCard extends React.Component {
 
   getCampaigns(staffPickOnly, paginate, page, callback) {
     this.requests.push(this.serverRequest = $.get(`${API_URL}campaigns?${staffPickOnly ? 'staff_pick=true&' : ''}${page != -1 ? 'page=' + page : ''}`, result => {
-
-      // Filter out campaigns that are closed, users have signed up for and then rank remaining campaigns
-      const campaigns = result.data
-        .filter(campaign => campaign.status !== 'closed')
-        .filter(campaign => this.campaignsToExclude.indexOf(parseInt(campaign.id)) == -1)
-        .sort(rankCampaigns);
+      // Get campaigns ordered by relevance & handle exclusions.
+      const campaigns = sortCampaignsByRelevance(result.data).filter(campaign => this.campaignsToExclude.indexOf(parseInt(campaign.id)) == -1);
```

Comment:
Yeah, potentially. Not needed for merging but something for later investigation ;)